docs/aio.md

```diff
@@ -50,57 +58,57 @@ interface AIOPin {
 callback ReadCallback = void (unsigned long value);
 ```
 
-API Documentation
------------------
-### AIO.open
-
-`AIOPin open(AIOInit init);`
+AIO API
+-------
+### aio.open(AIOInit)
+* 'AIOInit' *object*  The AIOInit object has a single field called "pin"
+  that represents the name of the pin (either an integer or a string,
+  depending on the board).
 
-The `init` object lets you set the pin number. You can either use a raw
+When setting the pin number, you can either use a raw
 number for your device or use the board support module such as
 [Arduino 101](./boards/arduino_101.md) or [K64F](./boards/frdm_k64f.md) to
 specify a named pin.
 
-Use the AIOPin object returned to read values from the pin.
-
-### AIOPin.read
+Returns an AIOPin object that be used to read values from the pin.
 
-`unsigned long read();`
+AIOPin API
+----------
+### pin.read()
 
-Returns the latest reading from the pin. Blocks until it gets the result.
+Returns the latest reading from the pin (an unsigned integer). Blocks until it gets the result.
 
-### AIOPin.readAsync
+### pin.readAsync(ReadCallback)
+* 'ReadCallback' *callback* User-provided callback function that takes
```

Comment:
If callback is like a type and the variable name is `ReadCallback`, let's call it `readCallback` instead to match case in other arguments. (I'm a little confused. :D)

Reply:
So I'll respond here; push the explanation to the other places it applies:

I inconsistently read the Buffer documentation and just got this wrong.

I've now edited the documents to fit the following rules (which match the examples in the WebIDL spec):

Variable names are camelCase, starting with a lower-case letter.
Type names are UpperCase.